composer/workflows/airflow_db_cleanup.py

```diff
@@ -319,37 +315,27 @@ def build_query(
     airflow_db_model,
     age_check_column,
     max_date,
-    keep_last,
-    keep_last_filters=None,
-    keep_last_group_by=None,
+    dag_id=None
```

Comment:
It took me a while to understand that `dag_id` is passed/used only for DagRun model. It is worth to have docstring for this method with some explanation

Reply:
Makes sense. Updating.

Reply:
PTAL on the docstring

Reply:
Btw it is not used only for DagRuns, fixed! Thanks!